component-template/{{ cookiecutter.slug }}/Makefile

```diff
@@ -0,0 +1,38 @@
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := all
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+DOCKER_CMD   ?= docker
+DOCKER_IMAGE ?= docker.io/projectsyn/commodore-component-test:latest
```

Comment:
Let's use the official upstream images of the tools, so we can get rid of our custom image again.